pkg/tempopb/tempo.proto

```diff
@@ -121,6 +122,15 @@ message SearchTagValuesResponse {
   repeated string tagValues = 1;
 }
 
+message TagValue {
```

Comment:
Would prefer to leverage existing structs, but think that isn't possible here. The OpenTelemetry attribute struct is not sufficient to convey the type information because it can't distinguish between string and keyword (status "ok", etc), and there is actually no wire format for any TraceQL structs yet.  So this very basic struct was created and we need to settle on the possible values for the type strings.